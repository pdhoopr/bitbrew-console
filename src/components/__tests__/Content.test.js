import { shallow } from 'enzyme';
import React from 'react';
import Content from '../Content';

test('renders DOM for some content', () => {
  const dom = shallow(<Content />);
  expect(dom).toMatchSnapshot();
});
