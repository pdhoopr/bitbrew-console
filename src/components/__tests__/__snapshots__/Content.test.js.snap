// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DOM for some content 1`] = `
.c0 {
  -webkit-flex: 1 0 auto;
  -ms-flex: 1 0 auto;
  flex: 1 0 auto;
}

<main
  className="c0"
/>
`;
